src/components/tree-view/stories/Doc.mdx

```diff
@@ -0,0 +1,130 @@
+import { Canvas, Meta, Story, Source, ArgTypes, Markdown } from '@storybook/blocks';
+import * as Stories from './tree.stories';
+import { TreeView } from '../TreeView';
+import { TreeViewItem } from '../TreeViewItem';
+
+<Meta of={Stories}/>
+
+# Tree View
```

Comment:
When dropping into "children" the dropped element seem to dissappear


https://github.com/user-attachments/assets/a616e2a3-4212-485b-a120-1abedede5950